scss/tech.scss

```diff
@@ -0,0 +1,33 @@
+.tech-border {
+    border: 1px solid $green;
+    border-radius: 5px;
+    height: 350px;
+    width: 200px;
+    
+    .tech-container {
+        align-items: center;
+        background-color: $green;
+        border-radius: 5px;
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        justify-content: flex-start;
+        left: 10px;
+        position: relative;
+        top: 10px;
+        width: 100%;
+
+        * {
+            margin: 10px 0;
+        }
+
+        :nth-child(2), :nth-child(3) {
```

Comment:
Apply the styles in another class. And apply them in the propertly tag
For example: `<p class='xxx'/>`

Reply:
Remember to be more generic with the definition of styles. In this case you can use this styles for childrens of an specific parent.